### DISCUSSION TITLE
Lab2-[Errno 110] Connection timed out

### DISCUSSION THREAD

**Discussion by yushliu** [bol-edu/HLS-SOC-Discussions#12] [Q&A] [answered]
When I tried to execute .ipynb file, it shows that the connection timed out.
Then the reference mentioned below reveals that's the problem of socket?
How can I solve this issue
Thanks!

reference: https://github.com/ipython/ipython/issues/3241/
![image](https://user-images.githubusercontent.com/92939085/190847469-17157675-e0d5-4e4a-a733-3a71321cd0f6.png)

**Comment by yushliu** [comment, marked as answer]
Solved
Try another FPGA board

**Comment by kevinjantw** [comment]
Hi, PYNQ community mentioned a same "[Errno 110] Connection timed out Error", it is likely a HW issue. You can try to find out the root cause from community discussion. https://discuss.pynq.io/t/timeouterror-errno-110-connection-timed-out-error/1972
